documentation/devonfw-ide-contribution-getting-started.asciidoc

```diff
@@ -0,0 +1,71 @@
+Find the license for the new tool and check if the license exists in the https://github.com/devonfw/ide/blob/master/documentation/LICENSE.asciidoc[document]. If the license does not exist, add it to the file and complete the table of contents.
+
+== Settings
+The https://github.com/devonfw/ide-settings[IDE settings] allow project-specific configurations of the IDE tools. For your own settings, fork the repository, adjust the configuration and specify the repository URL during installation. Important notes on configuration and especially configuration files can be found https://devonfw.com/website/pages/docs/devonfw-guide_ide.wiki_configuration.asciidoc.html[here]. In the configuration files many variables can be defined, listed https://github.com/devonfw/ide/blob/master/documentation/variables.asciidoc[here].
+
+== Tipps and tricks
+
+=== Bash (Linux's Borne Again Shell)
+**For beginners:**
+
+You probably know this black box on Windows (the Command Prompt) that accepts commands like 'dir' that shows the content of the directory you are currently in,
+or 'cd' with which you can change the directory you are currently in and many more commands that allow you to interact with your Windows Operating System in a more direct and eventually faster way.  
+Now for users of a Linux Distribution there is something similar, but way more powerfull, usually called "Bourne Again Shell", or in short Bash. This Command Line Interface or "Shell" is what we use most often when implementing something new for our Command-Line-Tool with the name devon-IDE.  https://www.youtube.com/watch?v=I4EWvMFj37g[Have a look at Bash in 150 Seconds.]
+For MacOS you often also use BASH. That's why it is often enough to implement code for Mac or Linux (at least to some extend as you'll see soon enough) to be able to use it on both Operating Systems.
+
+Let's get to the practical part of the Bash-usage. 
+
+**If you are using Windows**, _then please install WSL so you will be able to use Bash as well. E.g. https://www.youtube.com/watch?v=qYlgUDKKK5A[watch this] guide untill Minute 5:58 for a basic setup on Windows and some command examples._
+
+On the internet are some really good ressources for Bash-beginners.
+That's what I now want to make use of by sharing https://www.youtube.com/watch?v=oxuRxtrO2Ag [this robust introduction], going through some of the most important commands and concepts in round about 70 Minutes (you can find a list of the commands introduced by Joe at the videos description). There are some really usefull commands, that I would have liked to know back then when I started with Bash. When you have learned the knowledge from this video, then you'll be a big step further on your journey with Bash. 
+I think it is really worth watching such a video and to directly play around with those commands on your own Shell simultaneously.  This makes the learning process faster and also way more interesting than simply reading or listenning to such a tutorial.  
```

Comment:
Sounds a little bit more like a personal review on the youtube videos (`I think ...`).
IMHO you could simple list a bullet point of links and add once sentence to give the reader an idea what to expect and if it is worth watching for him or not.

Reply:
The advantage of that would definitely be a fast overview of the resources mentioned. 
On the other side I wanted to accompany the reader a bit more, by not being to short.  Especially for people that are not so familiar with bash-coding yet this would be more comfortable to read and to process. I would also not say, that this particular video is definitely the best around the web (and I'm happy for other sources, that may fit our needs in a better way), but the producer seems to have a good approach for beginners.
Still, if you or the others think that it is to much, then I'll be fine to just make a list of the resources with some brief notes next to them.